src/linker/Linker/TypeMapInfo.cs

```diff
@@ -166,7 +167,8 @@ void MapVirtualMethods (TypeDefinition type)
 				return;
 
 			foreach (MethodDefinition method in type.Methods) {
-				if (!method.IsVirtual)
+				// This could be a static interface method implementation
+				if (!method.IsStatic && !method.IsVirtual)
```

Comment:
```suggestion
				if (method.IsStatic || !method.IsVirtual)
```

I find this one easier to read because there's less negation. Also, it can be evaluated faster.